core/test/financial_templates/ExpiringMultiParty.js

```diff
@@ -1,29 +1,36 @@
+const { toWei } = web3.utils;
+
+// Tested Contract
 const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
-const Finder = artifacts.require("Finder");
+
 // Helper Contracts
+const Finder = artifacts.require("Finder");
 const ERC20MintableData = require("@openzeppelin/contracts/build/contracts/ERC20Mintable.json");
 const truffleContract = require("@truffle/contract");
 const ERC20Mintable = truffleContract(ERC20MintableData);
 ERC20Mintable.setProvider(web3.currentProvider);
 
 contract("ExpiringMultiParty", function(accounts) {
-  it("Empty", async function() {
-    finder = await Finder.new();
+  it("Can deploy", async function() {
+    const collateralToken = await ERC20Mintable.new({ from: accounts[0] });
+    const finder = await Finder.new();
 
-    const collateralAddress = await ERC20Mintable.new({ from: accounts[0] });
-    const { toWei } = web3.utils;
-    await ExpiringMultiParty.new(
-      true,
-      "1234567890",
-      "1000",
-      collateralAddress.address,
-      { rawValue: toWei("1.5") },
-      { rawValue: toWei("0.1") },
-      { rawValue: toWei("0.1") },
-      { rawValue: toWei("0.1") },
-      1000,
-      finder.address,
-      web3.utils.utf8ToHex("TESTUMA")
-    );
+    const constructorParams = {
+      isTest: true,
+      expirationTimestamp: "1234567890",
+      withdrawalLiveness: "1000",
+      collateralAddress: collateralToken.address,
+      finderAddress: finder.address,
+      priceFeedIdentifier: web3.utils.utf8ToHex("UMATEST"),
+      syntheticName: "Test UMA Token",
+      syntheticSymbol: "UMATEST",
+      liquidationLiveness: "1000",
+      collateralRequirement: { rawValue: toWei("1.5") },
+      disputeBondPct: { rawValue: toWei("0.1") },
+      sponsorDisputeRewardPct: { rawValue: toWei("0.1") },
+      disputerDisputeRewardPct: { rawValue: toWei("0.1") }
+    };
+    await ExpiringMultiParty.new(constructorParams);
+    assert(true);
```

Comment:
Do we need this assert here? I.e., if the deployment fails, would something already fail or no?

Reply:
I think you right; it's not required.